### ISSUE TITLE
Remember more settings

### ISSUE THREAD

**Issue by mifi** [mifi/lossless-cut#371]
- [ ] Discard audio
- [ ] Rotation

**Comment by tarve** [comment]
Rotation lock.

**Comment by mifi** [comment]
next version has a new jpeg quality settting

**Comment by hulleyrob** [comment]
I am constantly changing the black_min_duration for black detect and minChange for scene detect.

Would love for it to either remember my last setting or have a settings option to override the defaults.

**Comment by hulleyrob** [comment]
Can I add duration (detect silence scenes) to the list above. (1.0 seems to work better than 2.0 for me)

**Comment by Mrnofish** [comment]
I've upvoted #890 but I'm not sure if that gets noticed, since it's been collapsed in this multi issue (which seems like an effective way to send these to the issue graveyard, BTW).

Since I don't see a surge of upvotes coming for that particular improvement, is there a way to disable the blur from the console or something the like ?

I'm not particular about a pretty switch, I just need to get rid of that nasty, nasty blur.

**Comment by mifi** [comment]
You could try to try to add a CSS rule. if you open developer tools and select the blur background, you could find the rule and disable it. then I think you could also create some javascript that you could paste into the console to add a global css rule to disable it

**Comment by hulleyrob** [comment]
> Can I add duration (detect silence scenes) to the list above. (1.0 seems to work better than 2.0 for me)

Is this included in the above as it doesnt seem to have its own checkbox and must be my most made change every time I use the program?

**Comment by mifi** [comment]
> Is this included in the above as it doesnt seem to have its own checkbox and must be my most made change every time I use the program?

yes it's part of the bullet point "Black/scene/silence detect dialog parameters etc"

**Comment by mifi** [comment]
Black/scene/silence detect dialog parameters will be remembered in the next version (until app is restarted)

**Comment by hulleyrob** [comment]
YAY! Been waiting for this. Thank you.

**Comment by hulleyrob** [comment]
Further to trying this out it's now obvious I open and close the window a lot between edits. Is there anyway to make this persist between sessions?

**Comment by mifi** [comment]
> Further to trying this out it's now obvious I open and close the window a lot between edits. Is there anyway to make this persist between sessions?

not exactly sure what you're asking and which version you're on

**Comment by hulleyrob** [comment]
> Black/scene/silence detect dialog parameters will be remembered in the next version (until app is restarted)

Is there any way for these settings to persist after closing and reopening the app. It's not possible in the current version I'm using. (3.64.0)

**Comment by mifi** [comment]
> Is there any way for these settings to persist after closing and reopening the app. It's not possible in the current version I'm using. (3.64.0)

not currently. you would have to write them down outside losslesscut and re-enter when you re-open the app after closing it

**Comment by hulleyrob** [comment]
> > Is there any way for these settings to persist after closing and reopening the app. It's not possible in the current version I'm using. (3.64.0)
> 
> 
> 
> not currently. you would have to write them down outside losslesscut and re-enter when you re-open the app after closing it

Thought so. It's just this is something I'm doing pretty much every time I use the app. I've managed to create keyboard shortcuts for everything else so that I can pretty much tap some keys to do what I want but entering this every time takes about as long as everything else. Please consider having a persistent setting for this even an ini override or something in the config file would be fine.

**Comment by mifi** [comment]
Is it possible to optimize your workflow so that you don’t have to close the app between each file you process?

**Comment by hulleyrob** [comment]
I tried and the first thing I did after export was close the app...

I can see this being a problem which is why I asked if it would be at all possible. 

I can see why you want to reset the defaults but if there was a way for someone to change those defaults on purpose that most people would not do that would be great.

**Comment by mifi** [comment]
Anything is possible but i need to prioritize, and i feel that storing it until app restart is a good compromise.

**Comment by mifi** [comment]
> - [x] [Save which segments are selected/unselected in .llc file #1329](https://github.com/mifi/lossless-cut/issues/1329) [Remember excluded clips (selected/deselected segments) #885](https://github.com/mifi/lossless-cut/issues/885)

next version will have this

**Comment by xeophyte** [comment]
Can you add an option in the settings to permanently enable/disable these options? For example, I never use keyframes and when they're enabled, these lines always interfere with my work. Showing the keyframes lines causes confusion on the bar and disorientation with the segments. I prefer clearer bar only showing the segments. This just look bad:

<img width="820" height="60" alt="Image" src="https://github.com/user-attachments/assets/b4320f79-9a55-44cb-81e4-c66a5d1aaa4c" />

**Comment by hulleyrob** [comment]
Lucky you, and here I am still changing silent detection duration from 2.0 to 1.0 every single launch...

**Comment by mifi** [comment]
> Lucky you, and here I am still changing silent detection duration from 2.0 to 1.0 every single launch...

do you launch the app very often? :)

**Comment by hulleyrob** [comment]
Sadly nearly every time I go to use the app!!!

I just have to many windows open and closing them makes using "stage manager" easier.

**Comment by xeophyte** [comment]
Add remembering of advanced settings switch in the settings.

**Comment by mifi** [comment]
> Add remembering of advanced settings switch in the settings.

fixed in 251117e9094628ec05c2f3a1917f22eb85c0b1ff
